.github/pull_request_template.md

```diff
@@ -51,6 +51,9 @@ and would like to solicit the team's feedback. Optionally, provide background
 information regarding your questions and concerns. -->
 
 ### Acceptance Validation
+- [ ] This branch has been deployed to the AWS `dev` environment
+<!-- If no, why does this change not break CI/CD?  How is it not affected by using a persistent
+  database? -->
 
 <!-- Were you able to fully test the acceptance criteria on the related ticket? if not, why not? -->
```

Comment:
The priority of the `Acceptance Validation` is ensuring that the acceptance criteria has been fully tested.  That being said, should this commented section be the first item in `Acceptance Validation`.  From there, the additional comments regarding reminding the developer to consider deploying to `dev` could follow.